src/tests/SimilarWeb.js

```diff
@@ -1,77 +1,80 @@
 import logger from '../logger.js';
 
-const test = 'Similarweb';
-
 /**
  * Retrieve the Similarweb rank for a given domain
- * @param {string} entryDomain The domain to check
+ * @param {string} domain The domain to check
  * @param {*} env The environment
- * @param {string} [file] The filename of the entry, should only be set for non-primary domains
- * @returns {Promise<number>} Returns `0` if it's a success, `1` otherwise
+ * @returns {Promise<string>} Returns SimilarWeb rank if found
+ * @throws {Object<message, title>} Throws error object if fetching fails or rank exceeds limit
  */
-export default async function (entryDomain, env, file) {
-	const domain = getBaseDomain(entryDomain);
-	const res = await fetch(`https://api.similarweb.com/v1/similar-rank/${domain}/rank?api_key=${env.SIMILARWEB_API_KEY}`, {
-		cf: {
-			cacheTtlByStatus: {
-				'200-299': 7 * 24 * 60, // Cache 7 days
-				'300-599': 60, // Cache 1 hour
-			},
-		},
-	});
-
-	if (res.status === 404) {
-		if (!file) throw new Error(`${domain} doesn't have a Similarweb rank.`);
-		else logger.addWarning(file, `Additional domain ${domain} doesn't have a Similarweb rank.`);
-	}
-
-	if (!res.ok) {
-		const errorDetails = `HTTP error ${res.status}\nError message: ${await res.text()}`;
-		if (!file) throw new Error(`Unable to fetch website rank — ${errorDetails}`);
-		else logger.addWarning(file, `Unable to fetch website rank for additional domain ${domain} — ${errorDetails}`);
-
-		return 1;
-	}
+export default async function(domain, env) {
+  const API_KEY = getAPIKey(env);
+  const res = await fetch(
+    `https://api.similarweb.com/v1/similar-rank/${domain}/rank?api_key=${API_KEY}`,
```

Comment:
```suggestion
  const apiKey = getAPIKey(env);
  const res = await fetch(
    `https://api.similarweb.com/v1/similar-rank/${domain}/rank?api_key=${apiKey}`,
```

Uppercase variables usually refer to constants or globals. Since this is a local variable, let's use camel case.